src/PageMeta.php

```diff
@@ -74,7 +74,7 @@ public function get(
         bool $siteFallback = false,
         bool $configFallback = false,
         mixed $fallback = null
-    ): Field {
```

Comment:
@srijan Please add a "to do" comment, so I can add the type check again for later.

Reply:
I've replaced the Field namespace in the use section instead. This, I think, makes it incompatible with v3.